contracts/mocks/TestnetFaucet.sol

```diff
@@ -107,12 +123,32 @@ contract TestnetFaucet is
         return IERC20Upgradeable(asset).balanceOf(address(this));
     }
 
+    function addAsset(address asset, uint256 amount) external override onlyOwner {
```

Comment:
what if asset already exists in array?

Reply:
I ignore this and assume we don't do such a stupid thing as adding twice the same asset.
It is on test net, so that I will add a test for this

Reply:
ok